ansible/roles/stack-sunbird/templates/learner-service_logback.xml

```diff
@@ -6,8 +6,25 @@
   <timestamp key="timestamp" datePattern="yyyy-MM-dd"/>
   <!-- common transactions logs -->
   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-    <encoder>
-      <pattern>%d %msg%n</pattern>
+    <encoder class="net.logstash.logback.encoder.LogstashEncoder">
+      <layout class="net.logstash.logback.layout.LogstashLayout">
+        <fieldNames>
+          <timestamp>timestamp</timestamp>
+          <message>msg</message>
+          <logger>lname</logger>
+          <thread>tname</thread>
+          <levelValue>[ignore]</levelValue>
+          <version>[ignore]</version>
+          <stack_trace>exception</stack_trace>
+        </fieldNames>
+        <throwableConverter class="net.logstash.logback.stacktrace.ShortenedThrowableConverter">
+          <maxDepthPerThrowable>30</maxDepthPerThrowable>
+          <maxLength>2048</maxLength>
```

Comment:
can we variablize these values @amit-tarento ?

Reply:
We can , But not required as its just to print exception stacktrace , with limited verbose.